src/handlers/metadataDelta/handlers/removeVerifiedMembers.ts

```diff
@@ -0,0 +1,70 @@
+import { mutate, query } from '~amplifyClient';
+import {
+  ColonyActionType,
+  DeleteVerifiedMemberDocument,
+  DeleteVerifiedMemberMutation,
+  DeleteVerifiedMemberMutationVariables,
+  GetVerifiedMemberDocument,
+  GetVerifiedMemberQuery,
+  GetVerifiedMemberQueryVariables,
+} from '~graphql';
+import { ContractEvent } from '~types';
+import { verbose, writeActionFromEvent } from '~utils';
+import {
+  MetadataDeltaOperationType,
+  MetadataDeltaOperation,
+  RemoveVerifiedMembersOperation,
+} from '../types';
+
+export const isRemoveVerifiedMembersOperation = (
+  operation: MetadataDeltaOperation,
+): operation is RemoveVerifiedMembersOperation => {
+  return (
+    operation.type === MetadataDeltaOperationType.REMOVE_VERIFIED_MEMBERS &&
+    operation.payload !== undefined &&
+    Array.isArray(operation.payload)
+  );
+};
+
+export const handleRemoveVerifiedMembers = async (
+  event: ContractEvent,
+  operation: MetadataDeltaOperation,
+): Promise<void> => {
+  if (!isRemoveVerifiedMembersOperation(operation)) {
+    verbose('Malformed RemoveVerifiedMembers operation: ', operation);
+    return;
+  }
+
+  const { contractAddress: colonyAddress } = event;
+  const { agent: initiatorAddress } = event.args;
+
+  await Promise.all(
+    operation.payload.map(async (userAddress) => {
+      const item = await query<
+        GetVerifiedMemberQuery,
+        GetVerifiedMemberQueryVariables
+      >(GetVerifiedMemberDocument, { colonyAddress, userAddress });
+
+      const verifiedMemberData = item?.data?.getVerifiedMember;
+
+      if (verifiedMemberData === null || verifiedMemberData === undefined) {
+        throw new Error(
+          `User ${userAddress} is not a verified member of the colony ${colonyAddress}`,
+        );
```

Comment:
Please make sure to wrap `Promise.all` in a try/catch block. Otherwise it'll be unhandled error that will cause ingestor to go into restart loop